LoboStore/src/main/java/com/loboevolution/store/laf/LAFSettings.java

```diff
@@ -190,7 +190,7 @@ private LAFSettings retriveFontDate() {
 				laf.setStrikethrough(rs.getInt(18) == 1);
 				laf.setFontSize(Float.parseFloat(rs.getString(19)));
 				laf.setFont(rs.getString(20));
-				laf.setColor(ColorFactory.getInstance().getColor(rs.getString(21)));
+				laf.setColor(Color.BLACK);
 				laf.setBold(rs.getInt(22) == 1);
```

Comment:
From the Tools/Preferences/Look & Feels tab it is possible to set the text color, with this fix is not more possible.